lib/charms/data_platform_libs/v0/data_interfaces.py

```diff
@@ -320,7 +320,7 @@ def _on_topic_requested(self, event: TopicRequestedEvent):
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 20
+LIBPATCH = 24
```

Comment:
Didi you tried to publish the library with version 24 manually? I am afraid that charmhub will not allow this.
Maybe before merging this PR we would need to manually publish version 21,22 and 23

Reply:
we dont own this lib, this was a charmcraft fetch